app/components/homepage-components/WorkExperienceCard.tsx

```diff
@@ -41,7 +64,7 @@ const WorkExperienceCard = ({ job }: WorkExperienceCardProps) => {
           {job.duration > 1 && "s"} Experience
```

Comment:
Great job! This works, but it can be made more explicit and readable using a utility function or a ternary operator.
Another approach to consider:
`const experienceUnit = job.duration > 1 ? "s" : "";`

And then you should be able to implement it as so:
`{job.position} - {job.duration} {job.durationUnit}{experienceUnit} Experience`

Again, this is just a suggestion. What you have is fine but this seems to be a cleaner, more readable approach.

Reply:
Great catch. Thanks. I need to get into the habit of running the logic before the return statement.